docs/docs-content/automation/palette-sdk/palette-sdk.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: "Palette Go SDK"
+title: "Palette Go SDK"
+description: "Learn how to get started with the Palette Go SDK."
+hide_table_of_contents: false
+tags: ["palette-sdk"]
+---
+
+The Palette Go Software Developer Toolkit (SDK) allows Go developers to interact with several public Palette APIs
+through user-friendly client configurations.
+
+The SDK comprises of two libraries:
```

Comment:
Sorry @caroldelwing... these have been merged. I should have given you a heads up earlier. Now there's only palette-sdk-go.

Reply:
No worries. Will remove the old reference :)